### DISCUSSION TITLE
I have my account linked with email, but project not show it.

### DISCUSSION THREAD

**Discussion by asukaminato0721** [community/community#40087] [Pull Requests]
### Select Topic Area

Bug

### Body

I have checked the manual https://docs.github.com/en/get-started/quickstart/contributing-to-projects, but there is no this topic.

Time sequence:

1. use my email as git email. (Forget to link it to my account.)
2. fork, make some commits.
3. PR, and get merged.
4. not appear in repo contributors.
5. link it to my account.
6. shown in my profile, but not in the repo contributors.

Example:

https://github.com/kmgill/mars-raw-utils/pull/28 this is my merged pr and shown in my profile, but my avatar not shown in project contributors.

**Comment by ossanna16** [comment]
Hi there @wuyudi and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
